wordpress-mobile/WordPress-iOS
# Fix crash in Sharing, adding `externalUsersOnly` property to `PublicizeService`

Fixes #9983

This PR reintroduces the `externalUsersOnly` property to `PublicizeService` in a new core data model version. See the original issue for some more information.

**To test:**

* Check that the Sharing menu item and functionality is working as expected
* If we can also double check that migrating from an earlier version of the app works as expected that'd be great. I _think_ we should be okay this time, as I created a new model version instead of updating the original one in place.
